browser/src/client_win.cc

```diff
@@ -28,19 +28,54 @@ ClientWin::ClientWin(Config config) : ClientBase(std::move(config)) {
     return;
 
   pipename_ = pipename;
+  if (ConnectToPipe(pipename_, &hPipe_) != ERROR_SUCCESS) {
+    Shutdown();
+  }
 }
 
-DWORD ClientWin::ConnectToPipe(HANDLE* handle) {
+int ClientWin::Send(const ContentAnalysisRequest& request,
+                    ContentAnalysisResponse* response) {
+  // TODO: avoid extra copy.
```

Comment:
What extra copy? From the context it seems to be `request` but I could be wrong.

Reply:
Sorry, no extra copy, just a copy.  But I wonder if we can avoid?  It might require changing the arg to `ContentAnalysisRequest request` instead of `const&`, and then doing `std::move()` here and at call site.  Same for comment below.  I will edit the comment to be clearer.